editions/tw5.com/tiddlers/filters/Mathematics Operators.tid

```diff
@@ -28,6 +28,9 @@ The mathematics operators take three different forms:
 ** <<.inline-operator-example "=1 =2 =3 =4 +[product[]]">>
 ** <<.inline-operator-example "=1 =2 =3 =4 +[average[]]">>
 
+
+! Complex calculus with operator combination
```

Comment:
Unless I'm misunderstanding the intent, it might be clearer to say "calculations" or "computations". Alternately we could change the heading to "Operation Combination", and chance the body from "Operators can be combined" to "Operations can be combined by concatenating them while merging the inner `][` characters".